### PR TITLE
feature/remove-symantec-provisioner

### PR DESCRIPTION
**Pull request by jrouzierinverse**
# Description

Remove Symantec Provisioner

# Delete branch after merge
YES

# NEWS file entries
## Enhancements
* Remove Symantec Provisioner

### CODE REVIEW

File: conf/pf.conf.defaults [hunk 1]
```diff
@@ -138,7 +138,7 @@ passthroughs=
 # fencing.proxy_passthroughs
 #
 # Comma-delimited list of domains to be use for apache passthrough
-proxy_passthroughs=crl.geotrust.com,ocsp.geotrust.com,crl.thawte.com,ocsp.thawte.com,crl.comodoca.com,ocsp.comodoca.com,crl.incommon.org,ocsp.incommon.org,crl.usertrust.com,ocsp.usertrust.com,mscrl.microsoft.com,crl.microsoft.com,ocsp.apple.com,ocsp.digicert.com,ocsp.entrust.com,srvintl-crl.verisign.com,ocsp.verisign.com,ctldl.windowsupdate.com,crl.globalsign.net,pki.google.com,www.microsoft.com,crl.godaddy.com,ocsp.godaddy.com,certificates.godaddy.com,crl.globalsign.com,secure.globalsign.com,cacerts.digicert.com,crt.comodoca.com,crl.incommon-rsa.org,crl.quovadisglobal.com,cert.incommon.org,crt.usertrust.com,crl.verisign.com,crl.starfieldtech.com,developer.apple.com,ts-crl.ws.symantec.com,certificates.intel.com,ocsp.sectigo.com
```
Comment (fdurand) [inline conf/pf.conf.defaults]: Not related to symantec provisionner i think

File: t/unittest/UnifiedApi/Controller/Config/Bases.t [hunk 2]
```diff
@@ -60,7 +60,7 @@ $t->options_ok("$base_url/advanced")
 $t->options_ok("$base_url/fencing")
   ->status_is(200)
   ->json_is('/meta/proxy_passthroughs/placeholder', undef)
-  ->json_is('/meta/proxy_passthroughs/implied', 'crl.geotrust.com,ocsp.geotrust.com,crl.thawte.com,ocsp.thawte.com,crl.comodoca.com,ocsp.comodoca.com,crl.incommon.org,ocsp.incommon.org,crl.usertrust.com,ocsp.usertrust.com,mscrl.microsoft.com,crl.microsoft.com,ocsp.apple.com,ocsp.digicert.com,ocsp.entrust.com,srvintl-crl.verisign.com,ocsp.verisign.com,ctldl.windowsupdate.com,crl.globalsign.net,pki.google.com,www.microsoft.com,crl.godaddy.com,ocsp.godaddy.com,certificates.godaddy.com,crl.globalsign.com,secure.globalsign.com,cacerts.digicert.com,crt.comodoca.com,crl.incommon-rsa.org,crl.quovadisglobal.com,cert.incommon.org,crt.usertrust.com,crl.verisign.com,crl.starfieldtech.com,developer.apple.com,ts-crl.ws.symantec.com,certificates.intel.com,ocsp.sectigo.com');
```
Comment (fdurand) [inline t/unittest/UnifiedApi/Controller/Config/Bases.t]: Not related to symantec provisionner i think

**Review by fdurand — CHANGES_REQUESTED** [review]

**Comment by fdurand** [comment]
We should have a upgrade script that remove or comment the symantec provisionner in the config

**Comment by jrouzierinverse** [comment]
> We should have a upgrade script that remove or comment the symantec provisionner in the config

It is here.
`addons/upgrade/to-13.0-remove-provisioner.pl`

**Review by satkunas — APPROVED** [review]
UI changes look good.

**Review by fdurand — APPROVED** [review]

**Comment by fdurand** [comment]
rebase and it will be merged

**Comment by jrouzierinverse** [comment]
> rebase and it will be merged

done

**Review by nqb — CHANGES_REQUESTED** [review]
There is documentation to remove under `docs`

**Comment by nqb** [comment]
I'm wrong, it's already done #7692

**Pull request merged**